src/components/modeler/Modeler.vue

```diff
@@ -1896,6 +1953,15 @@ export default {
         this.redirect(redirectUrl);
       }
     });
+
+    // AI Setup
+    this.currentNonce = localStorage.currentNonce;
+
+    if (window.ProcessMaker?.modeler?.process?.id) {
```

Comment:
Why are you implementing getPromptSessionForUser and getPromptSessionFroProcess?